scripts/partition_manager/partition_manager.rst

```diff
@@ -328,6 +332,90 @@ It includes :file:`autoconf.h` (which is generated by Kconfig) and uses a Kconfi
 
    #endif /* CONFIG_SOC_NRF9160 */
 
+.. _pm_regions:
+
+Regions
+=======
+The Partition Manager places partitions in different on *regions*.
+Each region is defined by a *start address*, a *size*, a *placement strategy* and conditionally a *device name*.
+Examples of regions are *internal flash* and *external flash*.
+A region only specifies a *device name* if there is a device driver associated with the region, for instance a driver for an external SPI flash.
+A partition sets its region using the ``region`` property in its configuration.
+The internal flash region is used if no region is specified.
+There are three types of *placement strategies*:
+
+#. Start to end
+   Place partitions in a start to end fashion.
+   Partitions stored in a region with this placement strategy can not affect its placement through the ``placement`` keyword.
+   The unused part of the region is exposed through a partition with the same name as the region.
+
+#. End to start
+   Place partitions in a end to start fashion.
+   Partitions stored in a region with this placement strategy can not affect its placement through the ``placement`` keyword.
+   The unused part of the region is exposed through a partition with the same name as the region.
+
+#. Complex
+   Place partitions according to their ``placement`` configuration.
+   The unused part of the region is exposed through a partition named ``app`` by default.
+   This value can be changed through a command line argument.
+
+Regions are defined in :file:`partition_manager.cmake` as shown below:
+
+.. code-block:: cmake
+
+  add_region(     # Define region without device name
+    otp           # Name
+    756           # Size
+    0xff8108      # Base address
+    start_to_end  # Placement strategy
+    )
+
+  add_region_with_dev(           # Define region with device name
+    flash_primary                # Name
+    ${flash_size}                # Size
+    ${CONFIG_FLASH_BASE_ADDRESS} # Base address
+    complex                      # Placement strategy
+    NRF_FLASH_DRV_NAME           # Device name
+    )
+
+External flash
+==============
+The Partition Manager supports external flash through the use of :ref:`pm_regions`.
+Any placeholder partition can specify that it should be stored in the external flash region.
+
+.. note::
+  Placeholder partitions stored in the external flash region can not specify a ``placement`` configuration.
+
+The external flash information is provided to the Partition Manager through three Kconfig options.
+
+Given the following code in the flash driver:
+
+.. code-block:: c
+
+   /* SNIP */
```

Comment:
Nit: Is `/* SNIP */` convention?

Reply:
No, and I guess people understand that this is not an entire C file xD removed it.